test/test_nn.py

```diff
@@ -1799,29 +1799,27 @@ def test_parameterlistdict_setting_attributes(self):
         self.assertTrue(len(w) == 0)
 
     def test_parameterlistdict_pickle(self):
+        # warning from torch.load call in _load_from_bytes used in UntypedStorage.__reduce__
+        regex = re.escape("You are using `torch.load` with `weights_only=False`")
```

Comment:
Would be nice to call it say `weights_only_regex` (and it does not need to be a regex if it's just a string, is it?)
```suggestion
        WEIGHTS_ONLY_WARN = "You are using `torch.load` with `weights_only=False`"
```